shell/platform/windows/flutter_windows_engine_unittests.cc

```diff
@@ -40,7 +42,9 @@ std::unique_ptr<FlutterWindowsEngine> GetTestEngine() {
 }
 }  // namespace
 
-TEST(FlutterWindowsEngine, RunDoesExpectedInitialization) {
+class FlutterWindowsEngineTest : public WindowsTest {};
```

Comment:
Just curious for my own knowledge, what are the benefits of this?

Reply:
The WindowsTest class has access to the dart code that is compiled into the test runner.  This tripped me up for a long time but you can't send and receive messages on windows unless you have a running engine, and you can't have a running engine without a dart kernel to execute.  (On iOS and Android that isn't the case.)  That's why I had to make this test suit a subclass of WindowsTest.